index.rst

```diff
@@ -266,6 +324,10 @@ This example assumes the component of interest is already in the ``STANDBY`` sta
     # Wait for script to execute
     await script.done()
 
+.. note::
+    
+    Providing the ``_init.yaml`` file (or any file with a ``_`` prefix) to the ``configurationOverride`` parameter will return an error.
```

Comment:
Please consider something along the lines of: "Specifying any file with a ``_`` prefix, such as ``_init.yaml``, will cause the `start` command to be rejected."

Reply:
I changed it but didn't quite use exactly what you said. Take a peek.